component/storage/couchdb/store.go

```diff
@@ -919,12 +947,19 @@ func createIndexes(db *kivik.DB, tagNamesNeedIndexCreation []string) error {
 
 func getQueryOptions(options []storage.QueryOption) storage.QueryOptions {
 	var queryOptions storage.QueryOptions
-	queryOptions.PageSize = 25
 
 	for _, option := range options {
 		option(&queryOptions)
 	}
 
+	if queryOptions.PageSize < 1 {
+		queryOptions.PageSize = 25
```

Comment:
Should page size be configurable?

Reply:
It is. 25 is just a default if the `spi.WithPageSize` option isn't used (or is used with an invalid value, like 0 or -1)